src/test/java/com/patientpal/backend/patient/service/PatientServiceTest.java

```diff
@@ -0,0 +1,187 @@
+import com.patientpal.backend.patient.dto.request.PatientProfileUpdateRequest;
+import com.patientpal.backend.patient.dto.response.PatientProfileResponse;
+import com.patientpal.backend.patient.repository.PatientRepository;
+import com.patientpal.backend.test.annotation.AutoKoreanDisplayName;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("NonAsciiCharacters")
+@AutoKoreanDisplayName
+class PatientServiceTest {
+
+    @Mock
+    private PatientRepository patientRepository;
+
+    @Mock
+    private MemberRepository memberRepository;
+
+    @InjectMocks
+    private PatientService patientService;
+
+    @Nested
+    class 간병인_프로필_생성 {
+
+        @Test
+        void 환자_프로필을_성공적으로_생성한다() {
+            // given
+            Member member = huseongRolePatient();
+            when(memberRepository.findByUsername(member.getUsername())).thenReturn(Optional.of(member));
+            when(patientRepository.save(any(Patient.class))).thenAnswer(invocation -> invocation.getArgument(0));
+            PatientProfileCreateRequest request = createPatientProfileRequest();
+
+            // when
+            PatientProfileResponse response = patientService.savePatientProfile(member.getUsername(), request);
+
+            // then
+            assertNotNull(response);
+            assertThat(response.getMemberId()).isEqualTo(member.getId());
+            assertThat(response.getName()).isEqualTo(request.getName());
+        }
+
+        @Test
+        void 환자_프로필을_생성할_때_권한이_없으면_예외가_발생한다() {
+            // given
+            Member member = huseongRoleCaregiver();
+            PatientProfileCreateRequest request = createPatientProfileRequest();
+            when(memberRepository.findByUsername(member.getUsername())).thenReturn(Optional.of(member));
+
+            // when & then
+            assertThatThrownBy(() -> patientService.savePatientProfile(member.getUsername(), request))
+                    .isInstanceOf(AuthorizationException.class);
+        }
+
+        //TODO 휴대폰 인증 진행 & 중복 가입 구현 후 테스트 완성
+//    @Test
+//    void 환자_프로필을_생성할때_중복가입이면_예외가_발생한다() {
+//        // given
+//        Patient patient = huseongRolePatient();
+//        when(memberRepository.findByUsername(patient.getMember().getUsername())).thenReturn(Optional.of(patient.getMember()));
+//        when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.of(patient));
+//        PatientProfileCreateRequest request = createPatientProfileRequest();
+//
+//        // when & then
+//        assertThatThrownBy(() -> patientService.savePatientProfile(patient.getMember().getUsername(), request))
+//                .isInstanceOf(EntityNotFoundException.class);
+//    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 70.